spec/markdiff/differ_spec.rb

```diff
@@ -346,5 +346,14 @@
         expect(subject.to_html).to eq '<div class="changed"><p><del class="del">a</del><ins>b</ins></p></div><ins><p>c</p></ins><ins><p>d</p></ins>'
       end
     end
+
+    context "with insertion and deletion on same positions" do
+      let(:after_string) { "JEG HEDDER Kurset give de studerende procesforståelse" }
+      let(:before_string) { "Kurset skal give de studerende procesforståelse" }
+
+      it "returns expected patched node" do
+        expect(subject.to_html).to eq '<ins>JEG</ins>Kurset <del class="del">skal</del><ins>HEDDER</ins> give de studerende procesforståelse'
```

Comment:
I think this expectation:

<ins>JEG</ins>Kurset <del class="del">skal</del><ins>HEDDER</ins> give de studerende procesforståelse

would be the following string 🤔 

> JEGKurset HEDDER give de studernde procesforståelse

but actual `after_string` is:

> JEG HEDDER Kurset give de studerende procesforståelse

Reply:
Yes it should, however that is an issue related to consecutive insertions being shown wrong, which I'll fix in https://github.com/r7kamura/markdiff/pull/6 as soon as I get the time. The issue that is fixed is this PR is that the diff (before this fix) was: 
```
<ins class=\"ins ins-before\">JEG</ins>Kurset <del class=\"del\">skal</del> give de studerende procesforståelse
```
That is, the insertion of HEDDER is excluded because it occurs on the same position as the deletion.

I think it'll make sense to finish up the other PR before merging this.

Reply:
Ah, I finally understand 👍